frontend/src/pages/modelServing/screens/metrics/BiasTab.tsx

```diff
@@ -1,38 +1,90 @@
 import React from 'react';
-import { PageSection, Stack, StackItem } from '@patternfly/react-core';
+import {
+  Bullseye,
+  PageSection,
+  Spinner,
+  Stack,
+  StackItem,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { useParams } from 'react-router-dom';
+import MetricsPageToolbar from '~/pages/modelServing/screens/metrics/MetricsPageToolbar';
+import BiasMetricConfigSelector from '~/pages/modelServing/screens/metrics/BiasMetricConfigSelector';
+import { BiasMetricConfig } from '~/concepts/explainability/types';
 import { useExplainabilityModelData } from '~/concepts/explainability/useExplainabilityModelData';
-import DIRGraph from './DIRChart';
-import MetricsPageToolbar from './MetricsPageToolbar';
-import SPDChart from './SPDChart';
-import EmptyBiasConfigurationCard from './EmptyBiasConfigurationCard';
+import TrustyChart from '~/pages/modelServing/screens/metrics/TrustyChart';
+import BiasMetricChartWrapper from '~/pages/modelServing/screens/metrics/BiasMetricChartWrapper';
+import { useBrowserStorage } from '~/components/browserStorage';
+import EmptyBiasConfigurationCard from '~/pages/modelServing/screens/metrics/EmptyBiasConfigurationCard';
+import EmptyBiasChartSelectionCard from '~/pages/modelServing/screens/metrics/EmptyBiasChartSelectionCard';
+
+const SELECTED_CHARTS_STORAGE_KEY_PREFIX = 'odh.dashboard.xai.selected_bias_charts';
+const OPEN_WRAPPER_STORAGE_KEY_PREFIX = `odh.dashboard.xai.bias_metric_chart_wrapper_open`;
+const BiasTab: React.FC = () => {
+  const { inferenceService } = useParams();
+
+  const { biasMetricConfigs, loaded } = useExplainabilityModelData();
+
+  const [selectedBiasConfigs, setSelectedBiasConfigs] = useBrowserStorage<BiasMetricConfig[]>(
+    `${SELECTED_CHARTS_STORAGE_KEY_PREFIX}-${inferenceService}`,
+    [],
+    true,
+    true,
+  );
+
+  if (!loaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
 
-const BiasTab = () => {
-  const { biasMetricConfigs } = useExplainabilityModelData();
   return (
     <Stack>
       <StackItem>
-        <MetricsPageToolbar />
+        <MetricsPageToolbar
+          leftToolbarItem={
+            <ToolbarItem>
+              <ToolbarItem variant="label">Metrics to display</ToolbarItem>
```

Comment:
I am not a fan of nested `ToolbarItem`, maybe you should make them a `ToolbarGroup` and use `Stack` inside it for the `ToolbarItem`. Some style nits, anyway.